nexus/src/app/sagas/region_snapshot_replacement_step_garbage_collect.rs

```diff
@@ -0,0 +1,233 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Clean up the volume that stashes the target replaced during a region
```

Comment:
By `Clean up` we mean `Delete` here?  If so, we should just say delete.

We could say:
Delete the temporary volume we created during the snapshot replacement step saga.

Maybe?

Reply:
Cool cool, done in 9a9cc609a